flight/targets/pipxtreme/fw/pios_board.c

```diff
@@ -164,43 +323,34 @@ void PIOS_Board_Init(void) {
 				}
 
 #endif /* PIOS_INCLUDE_PPM */
-				break;
-			}
-		case HWTAULINK_PPMPORT_DISABLED:
-		default:
-			break;
-	}
+        break;
+    }
+    case HWTAULINK_PPMPORT_SPORT:
```

Comment:
@mlyle you might have a thought about using the shared system to do this part. note that this is a slightly different module that is attached to the SPORT than normal but it uses the same ```PIOS_COM_FRSKY_SPORT``` variable